packages/suite-data/src/guide/index.ts

```diff
@@ -1,28 +1,40 @@
+import { join } from 'path';
 import * as fs from 'fs-extra';
 import simpleGit from 'simple-git';
-import { TMP, GB_SOURCE, GB_REVISION } from './constants';
+import { TMP, GB_SOURCE, GB_REVISION, DESTINATION, ASSETS_DIR } from './constants';
 import { parse } from './parser';
+import { transform } from './transformer';
 
 /**
  * Clears the TMP folder and then populates it with content of the
  * GB_SOURCE repository checked out at GB_REVISION commit.
  */
 const fetchSource = async () => {
-    fs.removeSync(TMP);
-    fs.mkdirpSync(TMP);
+    pruneDirectory(TMP);
     // Run all subsequent git commands in the TMP directory.
     const git = simpleGit({ baseDir: TMP });
     await git.clone(GB_SOURCE, '.');
     await git.checkout(GB_REVISION);
 };
 
+/** Ensures the given directory exists and prunes its contents. */
+const pruneDirectory = (path: string) => {
+    fs.removeSync(path);
+    fs.mkdirpSync(path);
+}
+
```

Comment:
I hate TS error here as you use pruneDirectory function before it is defined.

Reply:
bbf7490735b915551c1c0a7de0e9599ca913c318